policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml

```diff
@@ -17,13 +17,12 @@ metadata:
       href: https://www.taxformfinder.org/forms/2023/2023-arkansas-form-ar1000d.pdf#page=1
```

Comment:
Leave a comment mapping the numeric filing statuses

Reply:
what's 5?

Reply:
Married filing separately on separate returns.